CHANGELOG.md

```diff
@@ -64,6 +66,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - `@cumulus/common/test-utils/inTestMode`
     - `@cumulus/common/util/setErrorStack`
 
+- **CUMULUS-1102**
+
+   - Refactors local API server code to allow for full erasing of local datastack (ES and DynamoDB).
+   - Adds optional parameters to the @cumulus/api bin serve to allow for launching the api without destroying the current data.
```

Comment:
Seems like this belongs in the "Added" section?

Reply:
sure.